test/development/app-dir/owner-stack-react-missing-key-prop/owner-stack-react-missing-key-prop.test.ts

```diff
@@ -0,0 +1,205 @@
+import { nextTestSetup } from 'e2e-utils'
+import { getRedboxSource, waitForAndOpenRuntimeError } from 'next-test-utils'
+
+// TODO: When owner stack is enabled by default, remove the condition and only keep one test
+const isOwnerStackEnabled =
```

Comment:
this whole thing would probably be easier to read if it was split into separate tests for with/without owner stacks. Will also make it easier to delete the non-owner stack case as well.

Reply:
sounds good, moved them into 2 separate files